### PR TITLE
Added curl logger for debugging request

### PR DESCRIPTION
**Pull request by iampgzp**

### CODE REVIEW

File: Classes/Network/FLEXNetworkCurlLogger.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+// FLEXCurlLogger.h
+// Instagram
+//
+// Created by Ji Pei on 07/27/16
+// Copyright 2004-present Facebook. All Rights Reserved.
```
Comment (ryanolsonk) [inline Classes/Network/FLEXNetworkCurlLogger.h]: Let's drop these copyright headers

File: Classes/Network/FLEXNetworkRecorder.m [hunk 2]
```diff
@@ -122,6 +123,9 @@ - (void)recordRequestWillBeSentWithRequestID:(NSString *)requestID request:(NSUR
 
         [self postNewTransactionNotificationWithTransaction:transaction];
     });
+
+    // generate the curl command for all request to debugger while networking debugging is enabled
+    NSLog(@"%@",[FLEXNetworkCurlLogger curlCommandString:request]);
```
Comment (ryanolsonk) [inline Classes/Network/FLEXNetworkRecorder.m]: This will create a lot of log spew. Let's remove and just expose through the interface.

**Comment by ryanolsonk** [comment]
This great!! Just two comments on the commit.

**Comment by iampgzp** [comment]
Let me know if this works. Thanks

**Comment by ryanolsonk** [comment]
Looks great. Thanks!

**Pull request merged**